WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -2,7 +2,28 @@
   <div class="container">
     <div class="navbar-brand">
       <a href="<%= root_path %>"><%= image_tag "wca_logo.svg" %></a>
-      <a href="<%= root_path %>"><span>World Cube Association</span></a>
+      <span>
+        <a href="<%= root_path %>">World Cube Association</a>
+        <br>
+        <a href="https://www.instagram.com/worldcubeassociation/" target="_blank" class="hide-new-window-icon" style="text-decoration:none;">
```

Comment:
Wow that's three long lines.
I have two requests for these social media links:
  - do not inline style on the tag, use CSS (you can probably add a `a.no-decoration` class somewhere).
  - use rails helpers, it will look much better:
```html
<%= link_to(icon("instagram"), "https://www.instagram.com/worldcubeassociation/", target: "_blank", class: "hide-new-window-icon no-decoration") %>
```